components/Projects/ProjectsList.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState, useMemo } from 'react';
+import useSWR from 'swr';
+import { Project } from './Project';
+import { ProjectSkeleton } from './ProjectSkeleton';
+
+export const ProjectsList = () => {
+  const fetcher = (...args: Parameters<typeof fetch>) =>
+    fetch(...args).then((res) => res.json());
+  const { data, error } = useSWR('/api/project', fetcher);
+  const [isLoading, setIsLoading] = useState(true);
+
+  // Generate array of specified length with random key value
+  const skeletonProjectsToLoad = Array.from({length: 10}, () => (Math.random() + 1).toString(36).substring(7));
+
+  useEffect(() => {
+    if(data) {
+      setTimeout(() => {
+        setIsLoading(false)
+      }, 1000);
+    }
```

Comment:
hi @JakubChorzepa could you please tell me why 1 sec delay for this? just curious

Reply:
I just wanted to avoid quick flick on page when data would be load fast